### DISCUSSION TITLE
Categorize for Multivendor or Using Group

### DISCUSSION THREAD

**Discussion by devomman** [inspireui/support#5346] [New Feature/Ideas Requests] [answered]
Hi, Please allow us to categorize for Multivendor Type or using multivendor group using this plugin https://wclovers.com/product/woocommerce-frontend-manager-groups-staffs/

**Comment by ecomdeveloperbd** [comment, marked as answer]
Group wise home screen design

**Reply by devomman** [reply]
Yes Groupwise

**Reply by mamunag8** [reply]
Add this feature

**Reply by genuinesourcecode** [reply]
This is nice Idea very good and agreed
